action/protocol/execution/evm/evm.go

```diff
@@ -85,34 +88,33 @@ func NewParams(raCtx protocol.RunActionsCtx, execution *action.Execution, stateD
 		contractAddr := common.BytesToAddress(contract.Bytes())
 		contractAddrPointer = &contractAddr
 	}
-	producer := common.BytesToAddress(raCtx.Producer.Bytes())
-
-	gasLimit := execution.GasLimit()
-	// Reset gas limit to the system wide action gas limit cap if it's greater than it
-	if gasLimit > raCtx.ActionGasLimit {
-		gasLimit = raCtx.ActionGasLimit
-	}
 
 	context := vm.Context{
 		CanTransfer: CanTransfer,
 		Transfer:    MakeTransfer,
 		GetHash:     GetHashFn(stateDB),
 		Origin:      executorAddr,
-		Coinbase:    producer,
+		Coinbase:    common.BytesToAddress(raCtx.Producer.Bytes()),
 		BlockNumber: new(big.Int).SetUint64(raCtx.BlockHeight),
 		Time:        new(big.Int).SetInt64(raCtx.BlockTimeStamp.Unix()),
 		Difficulty:  new(big.Int).SetUint64(uint64(50)),
-		GasLimit:    gasLimit,
+		GasLimit:    execution.GasLimit(),
```

Comment:
why here does not need the adjusted gasLimit below?

Reply:
Good catch, it's bug. I forgot to change it back